docs/source/markdown/podman-run.1.md

```diff
@@ -1454,6 +1463,36 @@ The **fuse-overlay** package provides a userspace overlay storage driver, otherw
 the **vfs** storage driver, which is diskspace expensive and does not perform well. slirp4netns is
 required for VPN, without it containers need to be run with the **--network=host** flag.
 
+## TROUBLESHOOTING
+
+### Rootless cgroups v2 CPU limits
+
+On some systems, setting CPU limits such as **--cpu-quota** will fail with an
+error similar to the following:
+
+    Error: opening file `cpu.max` for writing: Permission denied: OCI runtime permission denied error
+
+This means that CPU limit delegation is not enabled for the current user. You
+can verify by running the following command:
+
+    cat "/sys/fs/cgroup/user.slice/user-$(id -u).slice/user@$(id -u).service/cgroup.controllers"
+
+Example output might be:
+
+    memory pids
+
+In the above example, `cpu` is not listed, which means the curent user does
+not have permission to set CPU limits.
+
+If you want to enable CPU limit delegation for all users, you can create the
+file `/etc/systemd/system/user@.service.d/delegate.conf` with the contents:
+
+    [Service]
+    Delegate=memory pids cpu io
+
+After logging out and loggin back in, you should have permission to set CPU
+limits.
+
```

Comment:
I think it is fine to have it documented only in `troubleshooting.md`

Reply:
Ok. What should be in the man pages? A note saying that if you have issues with setting limits you should check troubleshooting.md?

Reply:
yes, I'd just add a pointer to the troubleshooting.md page to avoid maintaining the same information in 3 different places

Reply:
Ok, that sounds good. Is troubleshooting.md hosted on the website or should I just link to https://github.com/containers/podman/blob/master/troubleshooting.md?